### PR TITLE
Increase event_property threshold

### PR DESCRIPTION
**Pull request by scholtzan**
I'm not sure if increasing this threshold has any implications?

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/event_types_history_v1/init.sql [hunk 1]
```diff
@@ -108,7 +108,7 @@ per_event_property AS (
   USING
     (category, event, event_property)
   WHERE
-    event_property_value_index <= 1000
+    event_property_value_index <= 5000
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/telemetry_derived/event_types_history_v1/init.sql:111]: Also open to setting a different threshold

**Review by wlach — CHANGES_REQUESTED** [review]
I'm not aware of any issues with increasing the threshold. However, I believe the right approach is to modify the templating here and regenerate the queries:

https://github.com/mozilla/bigquery-etl/blob/821f9d45dc79c869bb3285c55ec1c7dcf0459fec/bigquery_etl/events_daily/query_templates/event_types_history_v1/templating.yaml#L12-L17

**Comment by scholtzan** [comment]
Right, right

**Review by wlach — APPROVED** [review]
r+ with the yaml changes to non-Firefox reverted

**Pull request merged**